dev-workflow-ui/webContent/view/notesDialog.xhtml

```diff
@@ -35,12 +39,12 @@ body .notes-paginator .ui-paginator-bottom {
   <ui:include src="/ivy-form/layouts/includes/progress-loader.xhtml" />
   <div id="content" class="container frame">
     <ui:insert name="content">
-      <h:form id="notesForm" styleClass="notes-form">
+      <h:form id="notesForm" styleClass="notes-form" rendered="#{notesBean.hasUserPermission()}">
```

Comment:
why not use an f:event and redirect to login page? e.g. as you do it on the task.xhtml?
`<f:event type="preRenderView" listener="#{tasksDetailsIvyDevWfBean.redirectIfCantResume}" />`